Common/Securities/DefaultMarginCallModel.cs

```diff
@@ -148,19 +149,19 @@ protected virtual SubmitOrderRequest GenerateMarginCallOrder(Security security,
             // compute the amount of quote currency we need to liquidate in order to get within margin requirements
             var deltaAccountCurrency = totalUsedMargin - totalPortfolioValue;
 
-            var currentlyUsedBuyingPower = security.BuyingPowerModel.GetReservedBuyingPowerForPosition(
-                new ReservedBuyingPowerForPositionParameters(security)).AbsoluteUsedBuyingPower;
+            var positionGroup = Portfolio.Positions.CreateDefaultGroup(security);
+
+            var currentlyUsedBuyingPower = positionGroup.BuyingPowerModel.GetReservedBuyingPowerForPositionGroup(Portfolio, positionGroup);
```

Comment:
Remember we talked about this case being a bit of a PITA. The value of `currentlyUsedBuyingPower` might not be accurate because it's using the `DefaultGroup` and this security could actually be part of another group/S determined by the resolver, right? Those groups could be accessed through the collection at the `PositionManager` right?